test/unit/pjax.js

```diff
@@ -32,6 +32,22 @@ if ($.support.pjax) {
     })
   })
 
+  asyncTest("copes with ampersands when pushing urls", function() {
```

Comment:
Maybe rename the test to something like "compatible with pathnames that contain ampersand"?